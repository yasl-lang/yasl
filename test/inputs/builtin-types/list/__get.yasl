const x = [.a, .b, .c]

echo x[0]
echo x[1]
echo x[2]

echo x->__get(0)
echo x->__get(1)
echo x->__get(2)

echo x[-1]
echo x[-2]
echo x[-3]

echo x->__get(-1)
echo x->__get(-2)
echo x->__get(-3)
