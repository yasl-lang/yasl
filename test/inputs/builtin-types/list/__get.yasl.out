a
b
c
a
b
c
c
b
a
c
b
a
